relay-server/src/extractors/request_meta.rs

```diff
@@ -363,6 +356,12 @@ pub type PartialMeta = RequestMeta<Option<PartialDsn>>;
 impl PartialMeta {
     /// Extracts header information except for auth info.
     fn from_headers<S>(request: &HttpRequest<S>) -> Self {
+        let mut ua = RawUserAgentInfo::default();
+
+        for (key, value) in request.headers() {
+            ua.extract_header(key.as_str(), value.to_str().ok().map(str::to_string));
```

Comment:
This solution is a bit inelegant, but since RawUserAgentInfo is defined in another crate I would have had to import Request library in General to make this an associated function. Since the other Header type is defined by us, we could perhaps implement From<HttpRequest> to make it prettier.